package.json

```diff
@@ -77,5 +77,5 @@
     "tiny-glob": "^0.2.9",
     "typescript": "^4.9.5"
   },
-  "packageManager": "pnpm@8.6.0"
+  "packageManager": "pnpm@8.0.0"
```

Comment:
aa Sorry to block on this, but I'm pretty sure this'll use `8.0.0` exactly 😅 

If you haven't tried it, try `pnpm@^8` or `pnpm@^8.0.0`

If _that_ still doesn't work, then we'll use exactly `8.6.12` and save the trouble 🫠

Reply:
`8.6.12` it is, the CI build blew up with `pnpm@^8.0.0` and the specs require a full `major.minor.patch` version 🙃